geps/gep-1748.md

```diff
@@ -0,0 +1,140 @@
+# GEP-1748: Gateway API Interaction with Multi-Cluster Services
+
+* Issue: [#1748](https://github.com/kubernetes-sigs/gateway-api/issues/1748)
+* Status: Experimental
```

Comment:
In just a general way I'm uncomfortable with things that would not _start_ in `Provisional` and spend some time there. Given how we use `Experimental` today, this would seem to indicate that most or all of the current questions and concerns about this initiative are more or less sorted out and we're ready to implement and try it. My suggestion would be:

```suggestion
* Status: Provisional
```

And then maybe to "experiment" (as distinct from what we call the "experimental" today) we could put it through the `Prototyping` status where we starting demoing an implementation? As far as I can see, this change in approach would result in a very similar trajectory, but with a more clear message that implementations should not necessarily start implementing this right away? I'm open to suggestion I just need some help to better understand the motivation.

Reply:
I think this is different than other GEPs in that we're not actually adding any APIs here or making any changes to existing APIs, we're just defining how APIs should interact. In that sense, I'd consider it similar to how Policy Attachment is also in an experimental state. 

> we could put it through the Prototyping status where we starting demoing an implementation?

There are already 2 implementations that support Gateway API and MCS API (GKE and AWS Lattice) with at least one more on the way soon. @liwenwu-amazon and I will be [speaking about our implementations at KubeCon](https://sched.co/1HyZO) in a couple weeks and I believe both of our implementations meet the guidelines described here. 

I can't really think of any other possible interpretation of how these APIs could interact with each other. In my opinion this is really just formalizing something that is already being done in practice.

Reply:
> In that sense, I'd consider it similar to how Policy Attachment is also in an experimental state.

Policy attachment has been a communicated concept for some time and something that's more of a suggestion in practice, so it seems a bit different from this which was only colloquially understood by some implementations implementing it independently, and is also somewhat niche.

> I can't really think of any other possible interpretation of how these APIs could interact with each other. In my opinion this is really just formalizing something that is already being done in practice.

I get this, and I personally agree with it which is why I said in my above review that I'm for the content overall. My main concern is that this comes in fast, we all think it's good, and then someone comes along later and actually does have some different interpretation but we're like "sorry we're in production with this already". We wouldn't want that scenario for anything, of course, I suppose the flip side is that we _always_ have the potential of running into this kind of problem even if things take a different path, if someone interested simply takes to long to show up :shrug: 

I'm getting the point that we're basically already in a scenario where "it's already a thing" as we're just formalizing something that people are already doing, so I perhaps the best thing for us to do is just make sure we get an approval from the 3 implementers we have so far and make sure we haven't missed anything. I'll consider this comment resolved thanks for the discussion.